packages/mdc-form-field/README.md

```diff
@@ -7,141 +7,95 @@ path: /catalog/input-controls/form-fields/
 
 # Form Fields
 
-MDC Form Field provides an `mdc-form-field` helper class for easily making theme-aware, RTL-aware
-form field + label combos. It also provides an `MDCFormField` class for easily making input ripples
-respond to label events.
+MDC Form Field aligns a form field (for example, a checkbox) with its label, and makes it RTL-aware and [theme](../mdc-theme)-aware.
+When used with JavaScript, it can activate a [ripple](../mdc-ripple) effect upon interacting with the label.
 
 ## Installation
 
 ```
 npm install @material/form-field
 ```
 
-## CSS Usage
+## Demos
 
-The `mdc-form-field` class can be used as a parent element, with any combination of adjacent `input` and `label` as
-immediate children:
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/checkbox.html">Demo with checkbox</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/radio.html">Demo with radio button</a>
+  </li>
+</ul>
 
-```html
-<div class="mdc-form-field">
-  <input type="checkbox" id="input">
-  <label for="input">Input Label</label>
-</div>
-```
+## Basic Usage
+
+### HTML Structure
 
-By default, this will position the label after the input. You can change this behavior using the
-`align-end` modifier class.
+The `mdc-form-field` class is used as a parent element, with any combination of adjacent `input` and `label` elements as
+immediate children:
 
 ```html
-<div class="mdc-form-field mdc-form-field--align-end">
+<div class="mdc-form-field">
   <input type="checkbox" id="input">
-  <label for="input">Input Label</label>
+  <label for="input">Label</label>
 </div>
 ```
 
-Now the label will be positioned before the checkbox.
-
-### Usage with MDC Web Components
-
-`mdc-form-field` works not just with `input` elements, but with _any_ immediate child element as long as its
-successive sibling is a `label` element. This means it will work for MDC Web form controls such as Checkbox and Radio:
+MDC Form Field works with _any_ type of immediate child element as long as its successive sibling is a `label` element.
+This means it will work for MDC Web form controls such as [MDC Checkbox](../mdc-checkbox) and [MDC Radio](../mdc-radio):
 
 ```html
 <div class="mdc-form-field">
   <div class="mdc-checkbox">
-    <input type="checkbox"
-           id="my-checkbox"
-           class="mdc-checkbox__native-control"/>
-    <div class="mdc-checkbox__background">
-      <svg class="mdc-checkbox__checkmark"
-           viewBox="0 0 24 24">
-        <path class="mdc-checkbox__checkmark-path"
-              fill="none"
-              stroke="white"
-              d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
-      </svg>
-      <div class="mdc-checkbox__mixedmark"></div>
-    </div>
+    ...
```

Comment:
I'd argue the inner content of the checkbox is somewhat important for correlating the `id` attribute of the input with the `for` atatribute of the label?